galaxyproject/galaxy
# [22.01] Add bioconductor as valid xref for tools

This adds an xref-tool type for bioconductor tools. The idea is to annotate tools like here: https://github.com/galaxyproject/tools-iuc/pull/4358

Also related to https://github.com/galaxyproject/galaxy/issues/13266

## How to test the changes?
(Select all options that apply)

With this the tools in https://github.com/galaxyproject/tools-iuc/pull/4358 should lint correctly.

## License
